pkg/cloudprovider/providers/vsphere/vclib/tagmanagers/tagmanegers.go

```diff
@@ -0,0 +1,32 @@
+package tagmanagers
+
+import (
+	"context"
+	"net/url"
+
+	"github.com/vmware/govmomi/vapi/tags"
+	"k8s.io/kubernetes/pkg/cloudprovider/providers/vsphere/vclib"
+)
+
+type TagManagers struct {
+	Client *tags.RestClient
+}
+
+func NewTagManagers(connection *vclib.VSphereConnection) TagManagers {
+	vsURL := connection.Client.URL()
+	vsURL.User = url.UserPassword(connection.Username, connection.Password)
+
+	return TagManagers{Client: tags.NewClient(vsURL, connection.Insecure, "")}
+}
+
+func (t TagManagers) WithClient(ctx context.Context, f func(client *tags.RestClient) error) error {
```

Comment:
Typo in file name: `tagmanegers.go`

Looks like this should just be a func in vclib, rather than a new package.  And longer term will go away when vmware/govmomi#1179 is fixed.

Reply:
Ok. So it's good that I move this file into `vclib` root floder and delete `tagmanagers` folder?

Reply:
`func withTagsClient` in vsphere.go should be fine.  No need for the `TagManagers` type.   If you do feel there's a need for the type, let's call it `tagsClient` instead of `TagManagers`.  And no need for a separate file as this function will go away when vmware/govmomi#1179 is fixed.